src/selectors/fridge.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2020
+ */
+
+import { createSelector } from 'reselect';
+import { UIDatabase } from '../database/index';
+import { chunk } from '../utilities/chunk';
+
+export const selectSelectedFridgeID = ({ fridge }) => {
+  const { selectedFridge = {} } = fridge;
+  const { id } = selectedFridge;
+
```

Comment:
Nit: role reversal 😆 ! Whitespace seems a bit redundant here?

Reply:
🤣 

trying to get a balance - I like a space before return although it's starting to get excessive